openrewrite/src/javascript/parserUtils.ts

```diff
@@ -1,4 +1,67 @@
 import * as ts from "typescript";
+import * as J from '../java'
+import * as JS from "./tree";
+
+export const statement_implementations_list: Array<{ new (...args: any[]): J.Statement }> = [
```

Comment:
Can we implement this as a type guard?

Reply:
Reimplemented the check as `isStatement` and `isExpression` methods

Unfortunately, there is no simple way in TS to require that the array contains only classes that implement the Statement/Expression interface.